_drafts/technical-interviews.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout:       post
+title:        Nailing the Technical Interview
+author:       gabby_losch
+summary:      Technical interviews don't need to be soulless. Here's the perfect answer, verified by science.
+image:        //res.cloudinary.com/wework/image/upload/v1459867978/technical-interviews.jpg
+categories:   engineering
+---
+
+### Standard Interviews
+
+If you've been on either side of a technical interview, you've probably heard questions about sorting algorithms, discussed Big O, asked or been asked how that function could be made faster, better, harder, stronger, and you've also likely wanted to run out of the room screaming. Or maybe you did run out of the room screaming. 
```

Comment:
general note, not just specifically here, but add links! (especially for "Big O"), could be informative to people wanting to learn, or could be used as a joke...